reactive/calico.py

```diff
@@ -370,6 +377,14 @@ def configure_calico_pool():
                 }
             }
 
+            if config['service-cluster-ips']:
+                pool['spec'].update({'serviceClusterIPs':
+                                    [config['service-cluster-ips']]})
+
+            if config['service-external-ips']:
+                pool['spec'].update({'serviceExternalIPs':
+                                    [config['service-external-ips']]})
+
```

Comment:
These are being set in the wrong place. They need to go in BGPConfiguration, not the IPPool. See https://docs.projectcalico.org/archive/v3.10/networking/advertise-service-ips.

Reply:
Sorry, fixed.